app/src/screens/IncomingCall/hooks.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { type Props as RemoteParticipantProps } from '../../components/Call/RemoteParticipant';
+
+export const useIncomingRemoteParticipant = () => {
+  const [title] = React.useState<string>('foo');
```

Comment:
Why not just set this to a const? Since you don't `setState`

Reply:
The function seems unnecessary?

Reply:
It's temporary, will be filled out with this ticket https://issues.corp.twilio.com/browse/VBLOCKS-1379